packages/worker/src/workflow/vm-shared.ts

```diff
@@ -265,7 +259,13 @@ export class GlobalHandlers {
           store.childToParent.delete(promise);
           store.promiseToStack.delete(promise);
         },
-      });
+      }) as () => void;
+    } catch (_) {
+      // v8.promiseHooks.createHook is not available in bun and Node.js < 16.14.0.
+      // That's ok, collecting stack trace is an optional feature anyway.
+      //
+      // FIXME: This should be sent to logs, not the console… but we don't have acces to it here.
```

Comment:
I appreciate the unicode ellipsis…

```suggestion
      // FIXME: This should be sent to logs, not the console… but we don't have access to it here.
```